pymc3/tests/test_transforms.py

```diff
@@ -85,18 +92,14 @@ def check_jacobian_det(
     actual_ljd = aesara.function([y], jac)
 
     computed_ljd = aesara.function(
-        [y], aet.as_tensor_variable(transform.jacobian_det(y)), on_unused_input="ignore"
+        [y], aet.as_tensor_variable(transform.jacobian_det(rv_var, y)), on_unused_input="ignore"
     )
 
     for yval in domain.vals:
         close_to(actual_ljd(yval), computed_ljd(yval), tol)
 
 
 def test_stickbreaking():
-    with pytest.warns(
-        DeprecationWarning, match="The argument `eps` is deprecated and will not be used."
-    ):
-        tr.StickBreaking(eps=1e-9)
```

Comment:
(Where) do we keep a list of these changes? We should mention them in the release notes. The alternative is to `raise` the `DeprecationWarning` which saves users from complicated digging.